versions17.txt

```diff
@@ -33,4 +33,5 @@
 1.7.6.3
 1.7.6.4
 1.7.6.5
+1.7.6.6
```

Comment:
Don't you think we can skip this version?

Reply:
We need to expose all our versions